resources/authgear/templates/en/web/authflowv2/__navbar.html

```diff
@@ -2,13 +2,13 @@
 <nav class="navbar">
   <h1 class="navbar__title">{{ $.Title }}</h1>
 
-  <a
-    class="navbar__item inline-flex items-center justify-center gap-x-3"
+  <button
+    class="navbar__item"
```

Comment:
When javascript is disabled, this button has no effect. So when javascript is disabled, it should be hidden. The idiomatic way to do this is to add `hidden` to the class, so initially it is hidden. When the controller connects, you run `this.element.classList.remove("hidden")` to make the button visible.

Reply:
Added this to https://github.com/authgear/authgear-server/issues/3626#:~:text=Always%20think%20about%20when%20the%20case%20when%20javascript%20is%20disabled

Reply:
Fixed up